cmd/endorse.go

```diff
@@ -180,16 +198,37 @@ func (f *endorseCommand) InitContext(ctx context.Context) (context.Context, erro
 	if err != nil {
 		return nil, fmt.Errorf("could not read UEFI file %s: %v", f.UefiPath, err)
 	}
+	if f.SvsmPath != "" {
+		ec.SvsmImage, err = ops.ReadFile(ctx, f.Storage, "", f.SvsmPath)
+		if err != nil {
+			return nil, fmt.Errorf("could not read SVSM file %s: %v", f.SvsmPath, err)
+		}
+		MeasurementTxt, err := ops.ReadFile(ctx, f.Storage, "", f.SvsmMeasurementPath)
+		if err != nil {
+			return nil, fmt.Errorf("could not read SVSM measurement file: %v", err)
+		}
+		const expectedMeasurementLength = 96
+		if len(MeasurementTxt) != expectedMeasurementLength+1 {
+			return nil, fmt.Errorf("SVSM measurement file must be %d hex characters and a newline", expectedMeasurementLength)
+		}
```

Comment:
```suggestion
		igvmMeasurementTxt = strings.Trim(igvmMeasurementTxt)
		igvmMeasurement, err := hex.DecodeString(string(igvmMeasurementTxt))
		if err != nil {
			return nil, fmt.Errorf("could not parse igvm measurement file %q: %v", igvmMeasurementPath, err)
		}
		if len(igvmMeasurement) != sgabi.MeasurementSize {
			return nil, fmt.Errorf("igvm measurement for SEV-SNP is %d bytes. Expected %d.", len(igvmMeasurement), sgabi.MeasurementSize)
		}
```
Where `import sgabi "github.com/google/go-sev-guest/abi"` is up top

Reply:
Thanks!